add_mq0_and_filter.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import sys
+import vcfpy
+import csv
+from collections import OrderedDict
+
+def to_array(dictionary):
+    array = []
+    for key, value in dictionary.items():
+        array.append("{}|{}".format(key, value))
+    return sorted(array)
+
+def parse_tsv_file(args):
+    values={}
+    with open(args.values_file,'r') as tsvin:
+        tsvin = csv.reader(tsvin, delimiter='\t')
+        for row in tsvin:
+            if any(x.strip() for x in row): #skip blank lines
+                values[(row[0] + ":" + row[1])] = row[2]
+    return values
+
+
+def create_vcf_reader(args):
+    vcf_reader = vcfpy.Reader.from_path(args.input_vcf)
+    if 'MQ0' in vcf_reader.header.format_ids():
+        print("FORMAT already contains a MQ0 field. It's value will be overwritten when matches are found")
+    if 'MQ0FRAC' in vcf_reader.header.format_ids() :
+        print("FORMAT already contains a MQ0FRAC field. It's value will be overwritten when matches are found")
```

Comment:
Or maybe even `Existing values will be overwritten when matches are found.` instead?
```suggestion
        print("FORMAT already contains a MQ0 field. Its value will be overwritten when matches are found.")
    if 'MQ0FRAC' in vcf_reader.header.format_ids() :
        print("FORMAT already contains a MQ0FRAC field. Its value will be overwritten when matches are found.")
```